src/NHibernate.Test/NHSpecificTest/NH1393/Fixture.cs

```diff
@@ -52,7 +52,7 @@ public void CanSumProjectionOnSqlFunction()
 					s.CreateCriteria(typeof (Person)).SetProjection(
 						Projections.Sum(Projections.SqlFunction(arithmaticAddition, NHibernateUtil.GuessType(typeof (double)),
 						                                        Projections.Property("IQ"), Projections.Property("ShoeSize"))));
-				IList list = c.List();
+				var list = c.List<object>();
```

Comment:
I think, for sake of "giving right examples", we should use a more specialized type. If you agree, I will add a revert commit then I will take the time of putting those more specialized types everywhere.

Reply:
Well I agree it makes sense in our `main` tests but I think it's not worth to update legacy bug related tests in NHSpecificTest folder. But I'm OK if you are willing to do it.

Reply:
Done. I have checked them with PostgreSQL and SQL-Server. Maybe some other db may fail.